app/src/components/Pages/SettingsPage.tsx

```diff
@@ -0,0 +1,258 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  Icon,
+  Stack,
+  Text,
+  theme as defaultTheme,
+  useDisclosure,
+  Wrap,
+} from '@chakra-ui/react';
+import { ipcRenderer } from 'electron';
+import React, { useEffect, useState } from 'react';
+import { IconType } from 'react-icons';
+import { FiBell, FiMail, FiTrash2 } from 'react-icons/fi';
+
+import { NotificationSettings } from '../../constants/types';
+import { logEvent } from '../../utils/analytics';
+import { STRIPE_LINK } from '../Premium/constants';
+import { useGoldContext } from '../Premium/GoldContext';
+import { EmailModal } from '../Support/EmailModal';
+
+function Notifications() {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [notifSettings, setNotifSettings] =
+    useState<NotificationSettings | null>(null);
+
+  useEffect(() => {
+    const fetchSettings = async () => {
+      setIsLoading(true);
+      const settings = await ipcRenderer.invoke('get-notification-settings');
+      setNotifSettings(settings);
+      setIsLoading(false);
+    };
+
+    fetchSettings();
+  }, []);
+
+  return (
+    <Box>
+      <Checkbox
+        size="lg"
+        colorScheme="purple"
+        onChange={(event) => {
+          const newNotifSettings = {
+            ...notifSettings,
+            responseRemindersEnabled: event.target.checked,
+          };
+
+          setNotifSettings(newNotifSettings);
+          ipcRenderer.invoke('set-notification-settings', newNotifSettings);
+        }}
+        disabled={isLoading}
+        isChecked={notifSettings?.responseRemindersEnabled}
+      >
+        Response Reminders
+      </Checkbox>
+    </Box>
+  );
+}
+
+function Application() {
+  return (
+    <Box>
+      <Button colorScheme="red">Reset Application</Button>
```

Comment:
```suggestion
      <Button colorScheme="gray">Refresh Data</Button>
```

I think it's confusing to have this called Reset Application if all it does is what the Refresh Data button does.

Also on that note, I think red is too scary, and we should make it gray. Because on the analytics page it's grey

Reply:
It technically does a little more, brings user all the way back. I think it should be a nuclear option.